### PR TITLE
Run several instances of solfuzzer in parallel

### PR DESCRIPTION
**Pull request by Marenz**

### CODE REVIEW

File: test/tools/afl_fuzzer.cpp [hunk 1]
```diff
@@ -93,18 +97,47 @@ Allowed options)",
...
-		FuzzerUtil::testStandardCompiler(input, quiet);
-	else
-		FuzzerUtil::testCompiler(input, !arguments.count("without-optimizer"), quiet);
+	bool optimize = !arguments.count("without-optimizer");
+	int retResult = 0;
+
+	for (string const& inputFile: inputs)
+	{
+		string input;
+		if (inputFile.size() == 0)
```
Comment (chriseth) [inline test/tools/afl_fuzzer.cpp:114]: It is a little awkward to see that inside the loop here. Is this actually reached?

Comment (Marenz) [reply test/tools/afl_fuzzer.cpp:114]: only if the file name was passed via std in

File: test/cmdlineTests.sh [hunk 2]
```diff
@@ -392,26 +392,9 @@ SOLTMPDIR=$(mktemp -d)
...
-        "$REPO_ROOT"/build/test/tools/solfuzzer --without-optimizer --quiet < "$f"
-        if [ $? -ne 0 ]
-        then
-            printError "Fuzzer (without optimizer) failed on:"
-            cat "$f"
-            exit 1
-        fi
-        set -e
-    done
+    echo *.sol | xargs -P 10 -n 50 "$REPO_ROOT"/build/test/tools/solfuzzer --quiet --input-files
```
Comment (chriseth) [inline test/cmdlineTests.sh]: I don't think `-P 10` is a good idea. This is usually run with at least one core at max already. I don't think we can assume more than 4 cores in most environments. I think the biggest speedup we get here is due to the `-n 50`.

**Comment by Marenz** [comment]
Changed amount of processes to 4

**Review by chriseth — APPROVED** [review]

**Review by chriseth — APPROVED** [review]

**Pull request merged**